pkg/descheduler/descheduler.go

```diff
@@ -140,9 +141,12 @@ func newDescheduler(rs *options.DeschedulerServer, deschedulerPolicy *api.Desche
 		return nil, fmt.Errorf("build get pods assigned to node function error: %v", err)
 	}
 
-	podEvictor := evictions.NewPodEvictor(
-		nil,
+	podEvictor, err := evictions.NewPodEvictor(
+		ctx,
+		rs.Client,
 		eventRecorder,
+		podInformer,
+		rs.DefaultFeatureGates,
```

Comment:
I was keeping in mind the case where the descheduling framework is run as a simulator. E.g. in case two or more framework instances are created in the same run/binary.